_test/test_data_loaders/test_rundata_rm_masked.m

```diff
@@ -35,22 +34,74 @@ function test_works_do_nothing(this)
             assertEqual(run.ERR,err);
             assertEqual(run.det_par,det);
         end
-        function test_works_removesNAN(this)
+        function test_works_removesNAN(~)
```

Comment:
Test should be renamed to say it removes INF and NaN by default, and an additional test created confirming that both are removed if `rm_masked(run, true, true);` is called would fully demonstrate the API